api/db-cdr/generate-cdr/validate-prerequisites-exist.sh

```diff
@@ -117,9 +135,22 @@ then
   exit 1
 fi
 
-# Purge all backup csv files except for the last 10 versions
 if [[ $CDR_VERSION != $PREP_TABLE_RUN ]];
 then
+
+  # Validate that all survey version exist
+  echo "Validating that all survey versions exist..."
+  q="select count(*) as count from \`$BQ_PROJECT.$BQ_DATASET.cb_survey_version\`
```

Comment:
nit: can we change the name of the query variable?

Reply:
Done.